### ISSUE TITLE
🚀 [Feature]: htmx support

### ISSUE THREAD

**Issue by gedw99** [gofiber/template#276]
### Feature Description

Htmx uses go templates or any templating system to send html to the browser client in a progressive enhancement way 

### Additional Context (optional)

https://news.ycombinator.com/item?id=32789004 Has links bout htmx and why golang coders are using it.

### Code Snippet (optional)

```go
package main

import "github.com/gofiber/template/%package%"

func main() {
  // Steps to reproduce
}
```

### Checklist:

- [X] I agree to follow Fiber's [Code of Conduct](https://github.com/gofiber/fiber/blob/master/.github/CODE_OF_CONDUCT.md).
- [X] I have checked for existing issues that describe my suggestion prior to opening this one.
- [X] I understand that improperly formatted feature requests may be closed without explanation.

**Comment by iredmail** [comment]
I use htmx (and hyperscript, alpinejs) too.

- official website: https://htmx.org
- documentation: https://htmx.org/docs/

**Comment by ReneWerner87** [comment]
htmx is something that is executed in the frontend

so how can we provide this as a template engine ?
can you please describe in more detail what you expect ?

**Comment by gedw99** [comment]
Well glad your asking 

here is a project that does it 

https://github.com/livefir/fir/tree/main

**Comment by gedw99** [comment]
yes i was thinking that we could add htmx as either an example or more integrated.

I am totally biased. I love htmx because it allows backend devs to do frontend without going nuts.

**Comment by gedw99** [comment]
> probably you want to have a workflow which combines the backend rendering process and active events from the frontend in one solution

YES exactly !!

**Comment by gedw99** [comment]
If you want to have a chat about it happy to. I am in Berlin. Saw your in germany

**Comment by ReneWerner87** [comment]
just join the discord channel and we can chat

best you create a feature request for the contrib repository and there you solve the combination of html template and sse/websocket in a middleware that sets up the application
https://github.com/gofiber/contrib/

**Comment by gedw99** [comment]
thanks @ReneWerner87 

wil make a feature request and join the chat

**Comment by gedw99** [comment]
Here is the new issue: https://github.com/gofiber/contrib/issues/679
